src/Core/Animation/Skeleton.cpp

```diff
@@ -144,38 +144,34 @@ void Skeleton::setTransform( const uint i, const Transform& T, const SpaceType M
         }
     }
     break;
-    default:
-    { CORE_ASSERT( false, "Should not get there" ); }
     }
 }
 
-int Skeleton::addBone( const int parent,
-                       const Transform& T,
-                       const SpaceType MODE,
-                       const Label label ) {
+uint Skeleton::addBone( const uint parent,
+                        const Transform& T,
+                        const SpaceType MODE,
+                        const Label label ) {
     switch ( MODE )
     {
     case SpaceType::LOCAL:
     {
         m_pose.push_back( T );
-        if ( parent == -1 ) { m_modelSpace.push_back( T ); }
+        if ( parent == uint( -1 ) ) { m_modelSpace.push_back( T ); }
```

Comment:
Seems strange to read this (`uint(-1)`). do you mean `0xFFFFFFFF` ? This reminds me a joke involving a genius that offer 3 wishes to a computer scientist and lost all its credibility ...